### PR TITLE
SEO-174654-Xamarine-Page-With-Redirect-One

### PR DESCRIPTION
**Pull request by Omanyadickson**
Hi @gowthamanthavasiyappan 

Kindly Review my PR below.
Regards
Dickson J

### CODE REVIEW

File: Xamarin/Picker/Overview.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: post
-title: About Xamarin Picker control | Syncfusion
+title: Overview About Xamarin Picker Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline Xamarin/Picker/Overview.md:3]: Update like "Overview of Xamarin Picker Control | Syncfusion"

File: Xamarin/Scheduler/overview.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: About Xamarin Scheduler control | Syncfusion
+title: Overview About Xamarin Scheduler Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline Xamarin/Scheduler/overview.md:2]: Change like "Overview of Xamarin Scheduler Control | Syncfusion"

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: Xamarin/Picker/Overview.md (hunk 1 above)
Comment (Omanyadickson) [reply Xamarin/Picker/Overview.md:3]: Resolved

File: Xamarin/Scheduler/overview.md (hunk 2 above)
Comment (Omanyadickson) [reply Xamarin/Scheduler/overview.md:2]: Resolved and updated it properly.

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**